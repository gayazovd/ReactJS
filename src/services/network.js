import axios from 'axios';

export const network = axios.create({
    baseURL: 'http://react-cdp-api.herokuapp.com',
});