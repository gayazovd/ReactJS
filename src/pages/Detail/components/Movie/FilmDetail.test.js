import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import FilmDetail from './index';
import { network } from '../../../../services/network';

configure({ adapter: new Adapter() });


describe('test <FilmDetail /> component', () => {
    let component;
    beforeEach(() => {
        component = shallow(<FilmDetail />);
    })

    it('<FilmDetail /> snapshot',() => {
        expect(component).toMatchSnapshot();
    }) 
    
})