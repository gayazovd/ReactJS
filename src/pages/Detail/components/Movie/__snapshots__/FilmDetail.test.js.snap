// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <FilmDetail /> component <FilmDetail /> snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
